recipes/asmjit/all/conanfile.py

```diff
@@ -68,8 +71,6 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
 
     def package_info(self):
-        self.cpp_info.names["cmake_find_package"] = "asmjit"
-        self.cpp_info.names["cmake_find_package_multi"] = "asmjit"
```

Comment:
Please keep it or use `set_property()` even if it matches the default, otherwise it's a nightmare to keep CCI recipes in good health regarding CMake nams.

Reply:
Oh, I misunderstood it.
I will revert it.